rosie/rosie.d.ts

```diff
@@ -134,11 +134,11 @@ declare namespace rosie {
      *   Factory.define('Person').sequence('id');
      *
      * @param {string} attr
-     * @param {Array.<string>=} dependencies
+     * @param {Array.<string>=} | Function dependencies or builder
      * @param {function(number): *=} builder
      * @return {Factory}
      */
-    sequence(name: string, dependencies?: string[], builder?: Function) : IFactory;
+    sequence(name: string, dependenciesOrBuilder?: Function | string[], builder?: Function) : IFactory;
```

Comment:
I feel that duplicating the documentation is a better choice here, as the overload style @mhegazy suggested is both more accurate and conveys intent more clearly. However since your addition matches the existing declarations, I'll merge this. I'll still encourage you to submit another PR that changes this and the existing declarations to overloads. Thank you!